src/main/java/org/folio/circulation/domain/representations/AccountStorageRepresentation.java

```diff
@@ -31,6 +33,8 @@ public AccountStorageRepresentation(Loan loan, Item item, FeeFineOwner feeFineOw
     this.put("loanId", loan.getId());
     this.put("userId", loan.getUserId());
     this.put("itemId", item.getItemId());
+    write(this, "dueDate", loan.getDueDate());
```

Comment:
Is it a new property recently introduced to the schema?

Reply:
No, it was always part of the Account entity in mod-feesfines but it was ignored in circulation.